Makefile

```diff
@@ -1,26 +1,13 @@
-# Copyright 2017 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-PLATFORM       ?= linux/amd64
```

Comment:
I think this enforcement is useful here since we may be building this locally (oftentimes on Arm Macs) for it to be used on prod Linodes (always linux/amd64). Or maybe let's think of it another way - linux/amd64 is the only supported platform, so we might as well only build artifacts with this platform, right?